### PR TITLE
Implement a MRS Prolog to drive utool.  Apply handle replacements from utool plug list of lists.  

### PR DESCRIPTION
**Pull request by MarcusGDaniels**

### CODE REVIEW

File: ws201.txt
Comment (arademaker) [inline ws201.txt:1]: I have deleted this file from the main branch. This was an old experiment to test the MRS parser. Please do not add it in the repo.

Comment (MarcusGDaniels) [reply ws201.txt:1]: Wasn't sure whether it was better to have clunky MRS data structures in code that are not easily reproducible or a file generated by Ace.

Comment (arademaker) [reply ws201.txt:1]: Sorry, I didn't understand. I don't think we need to save concrete examples in the repo, at least not in the root folder. In the future, we may add tests to a proper location.

Comment (MarcusGDaniels) [reply ws201.txt:1]: I removed the ws201.txt and use the run_ace to generate the MRS data structure now.

**Pull request closed without merging**